src/doc/en/installation/conda.rst

```diff
@@ -133,16 +102,29 @@ suffices to restart Sage.
 
 After editing any Cython files, rebuild the Sage library using::
 
-  $ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable src
+.. code-block:: shell
+
+    $ pip install --no-build-isolation --config-settings editable_mode=compat -v -v --editable src
 
 In order to update the conda environment later, you can run::
 
-  $ mamba env update --file environment-3.11-linux.yml --name sage-dev
+.. code-block:: shell
+
+    $ conda env update --file environment-3.12-linux.yml
+
+If you want to use Jupyter notebooks, you need to install JupyterLab and the SageMath kernel::
```

Comment:
I'm not sure if it's possible to do this, but if it is could you also explain how to create a Jupyter kernel for use in an existing/system Jupyter installation? This is something I've had trouble doing in conda so if there's a way to do it it would be nice to document.

Reply:
That's more complicated. You need to wrap things into an activation script and it's going to end up being more brittle. I am not sure whether I would want to advertise this here. (And I also don't want to test whether these instructions really work on Linux/macOS or even WSL.)

Reply:
Fair enough.